doc/PortingGuide.md

```diff
@@ -61,85 +61,113 @@ Porting Step by Step
 - add the qualifier `const` to each parameter which is not changed inside the
   kernel
 
-**CUDA kernel**
-```C++
-template< int blockSize >
-__global__ void fooKernel( int * ptr, float value )
-{
-    // ...
-}
-```
-
-**cupla kernel**
-```C++
-template< int blockSize >
-struct fooKernel
-{
-    template< typename T_Acc >
-    ALPAKA_FN_ACC
-    void operator()( T_Acc const & acc, int * const ptr, float const value) const
-    {
-        // ...
-    }
-};
-```
+  **CUDA kernel**
```

Comment:
yes this is what I want. I found the document confusing to read, because the code blocks weren't indented like the list. (in rendered form) It seemed to me like the list ended and I wondered what all these code blocks meant outside of the list and so on ...